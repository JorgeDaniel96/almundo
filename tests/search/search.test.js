import "react-native";
import React from "react";
import renderer from "react-test-renderer";
import Search from "~/src/components/search/search";

it("Search render is correct", () => {
  const tree = renderer.create(<Search />);
  expect(tree).toMatchSnapshot();
});
