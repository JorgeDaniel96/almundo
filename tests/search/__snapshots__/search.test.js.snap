// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search render is correct 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#f1f1f1",
        "paddingHorizontal": 10,
        "paddingVertical": 15,
        "width": 750,
      },
    ]
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../assets/images/iconSearch.png",
      }
    }
    style={
      Object {
        "left": "6%",
        "position": "absolute",
        "top": 25,
        "zIndex": 1,
      }
    }
  />
  <TextInput
    allowFontScaling={true}
    onChange={[Function]}
    onChangeText={[Function]}
    placeholder="Search hotel"
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "rgba(0, 0, 0, 0.08)",
        "borderRadius": 3,
        "borderWidth": 1,
        "fontSize": 14,
        "height": 42,
        "paddingLeft": "15%",
        "paddingRight": "25%",
        "width": "100%",
      }
    }
    underlineColorAndroid="rgba(0, 0, 0, 0)"
  />
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
        "position": "absolute",
        "right": "6%",
        "top": 25,
        "zIndex": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Image
      source={
        Object {
          "testUri": "../../../assets/images/iconClose.png",
        }
      }
    />
  </View>
</View>
`;
